sktime/forecasting/tests/test_ets.py

```diff
@@ -0,0 +1,42 @@
+__author__ = ["Hongyi Yang"]
+
+from numpy.testing import assert_array_equal
+from sktime.forecasting.ets import AutoETS
+from sktime.datasets import load_airline
+
+
+# test results against R implementation on airline dataset
+def test_airline():
+    y = load_airline().astype('float64')
+
+    # Default condition
+    # fit <- ets(AirPassengers, model = "ZZZ")
+    # components: "M" "A" "M" "TRUE" (error, trend, season, damped)
+    # discrepancy lies in damped (True in R but False in statsmodels)
+    fit_result_R = ['mul', 'add', 'mul']
+
+    forecaster = AutoETS(autofitting=True, sp=12, n_jobs=-1)
+    forecaster.fit(y)
+    fitted_forecaster = forecaster._fitted_forecaster
+    fit_result = [fitted_forecaster.error,
+                  fitted_forecaster.trend,
+                  fitted_forecaster.seasonal]
+
+    assert_array_equal(fit_result_R, fit_result)
+
+    # Allow multiplicative trend
+    # fit <- ets(AirPassengers, model = "ZZZ",
+    # allow.multiplicative.trend = TRUE)
+    # components: "M" "M" "M" "TRUE"
+    # discrepancy lies in damped (True in R but False in statsmodels)
+    fit_result_R = ['mul', 'mul', 'mul']
+
+    forecaster = AutoETS(autofitting=True, sp=12, n_jobs=-1,
+                         allow_multiplicative_trend=True)
+    forecaster.fit(y)
+    fitted_forecaster = forecaster._fitted_forecaster
+    fit_result = [fitted_forecaster.error,
+                  fitted_forecaster.trend,
+                  fitted_forecaster.seasonal]
+
+    assert_array_equal(fit_result_R, fit_result)
```

Comment:
I'd make this a separate function/test, makes it easier to debug if one of them fails